### ISSUE TITLE
TypeError: Cannot read property 'localeCompare' of undefined

### ISSUE THREAD

**Issue by cullylarson** [vercel/vercel#1428]
I'm getting the following error whenever I run `now`, but the deploy seems to work just fine (it starts up and runs as expected). Let me know if you need any more information from me to help diagnose the issue.

```
...
> Initializing…
> Building
> Error! An unexpected error occurred!
  TypeError: Cannot read property 'localeCompare' of undefined
    at exports.compare (/snapshot/repo/dist/now.js:6324:19)
    at module.exports.EventEmitter.onLog.setTimeout.buf.sort (/snapshot/repo/dist/now.js:11493:31)
    at Array.sort (native)
    at Timeout.module.exports.EventEmitter.onLog.setTimeout [as _onTimeout] (/snapshot/repo/dist/now.js:11493:16)
    at ontimeout (timers.js:478:11)
    at tryOnTimeout (timers.js:302:5)
    at Timer.listOnTimeout (timers.js:262:5) TypeError: Cannot read property 'localeCompare' of undefined
    at exports.compare (/snapshot/repo/dist/now.js:6324:19)
    at module.exports.EventEmitter.onLog.setTimeout.buf.sort (/snapshot/repo/dist/now.js:11493:31)
    at Array.sort (native)
    at Timeout.module.exports.EventEmitter.onLog.setTimeout [as _onTimeout] (/snapshot/repo/dist/now.js:11493:16)
    at ontimeout (timers.js:478:11)
    at tryOnTimeout (timers.js:302:5)
    at Timer.listOnTimeout (timers.js:262:5)
```

**Comment by jamo** [comment]
Try updating to latest `now-cli`.
Depending on how you installed it, it might auto-update or then you'll have to `npm i -g now`


Which version of `now` do you have? `now -v`

**Comment by cullylarson** [comment]
@jamo I just installed it using the macos installer. It's version `10.2.1`.

**Comment by alexpriceco** [comment]
I'm also having this error. Updating now didn't do it, but here's my log:
```
> Initializing…
> Building
> Error! An unexpected error occurred!
  TypeError: Cannot read property 'localeCompare' of undefined
    at exports.compare (/snapshot/repo/dist/now.js:6231:19)
    at module.exports.EventEmitter.onLog.setTimeout.buf.sort (/snapshot/repo/dist/now.js:11393:31)
    at Array.sort (native)
    at Timeout.module.exports.EventEmitter.onLog.setTimeout [as _onTimeout] (/snapshot/repo/dist/now.js:11393:16)
    at ontimeout (timers.js:478:11)
    at tryOnTimeout (timers.js:302:5)
    at Timer.listOnTimeout (timers.js:262:5) TypeError: Cannot read property 'localeCompare' of undefined
    at exports.compare (/snapshot/repo/dist/now.js:6231:19)
    at module.exports.EventEmitter.onLog.setTimeout.buf.sort (/snapshot/repo/dist/now.js:11393:31)
    at Array.sort (native)
    at Timeout.module.exports.EventEmitter.onLog.setTimeout [as _onTimeout] (/snapshot/repo/dist/now.js:11393:16)
    at ontimeout (timers.js:478:11)
    at tryOnTimeout (timers.js:302:5)
    at Timer.listOnTimeout (timers.js:262:5)
```

**Comment by leo** [comment]
This seems to be fixed since Now CLI 12.0.0!
